### PR TITLE
Add FMU test case

### PR DESCRIPTION
**Pull request by nllong**
I realize that this branch was called update-boptest, but this doesn't really update all of BOPTEST.

Updates:
* Add controller integration test case with an FMU model
* Increase max upload size to GitHub to 50MB which should only be temporary. The weather data within the FMU is quite large (40MB) due to the data being 5-minute data
* Update copyrights to now.year.
* Add thermal_comfort calculation library
* Add unit conversion library (with only k to c for now)
* Add manage task for pulling in latest boptest code that matters. Note that the latest code from BOPTEST breaks alfalfa and needs to be carefully integrated (or abstracted). `python setup.py update_boptest`

### CODE REVIEW

File: alfalfa_web/components/App/App.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ class App extends React.Component {
           <div className={this.props.classes.root}>
             <AppBar position="static">
               <Toolbar>
-                <Link to={'/'} className={this.props.classes.title} style={{ textDecoration: 'none', color: 'unset' }}> 
+                <Link to={'/'} className={this.props.classes.title} style={{ textDecoration: 'none', color: 'unset' }}>
```
Comment (nllong) [inline alfalfa_web/components/App/App.js:67]: The changes to the JS files are just the copyright year and removing any dangling spaces.

File: alfalfa_worker/lib/data/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+This directory contains modules to aid in the generation and management of
```
Comment (nllong) [inline alfalfa_worker/lib/data/README.md:1]: copied from latest boptest.

File: alfalfa_worker/lib/thermal_comfort.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# Copyright 2019
+# Author: Nicholas Long
+# This class calculates the predicted mean vote and percent people dissatisfied based on the methodology in
+# ASHRAE Standard 55 Normative Appendix B and in accordance with ISO 7730.
+
+import math
+
+
+class ThermalComfort(object):
```
Comment (nllong) [inline alfalfa_worker/lib/thermal_comfort.py:9]: new method to calculate thermal comfort based on ASHRAE Std 55.

File: alfalfa_worker/requirements/production.txt [hunk 4]
```diff
@@ -14,7 +14,8 @@ jmespath==0.9.4
 mlep==0.1.dev2
 more-itertools==5.0.0
 nodeenv==1.3.4
-pandas==0.24.2
+numpy==1.16.6  # need older version until Python > 3.5
```
Comment (nllong) [inline alfalfa_worker/requirements/production.txt:17]: need to upgrade to > Python 3.5. However, we can't do this until we upgrade away from Ubuntu 16.0x and ideally the dual python versions.

File: management/update_boptest.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+class UpdateBoptest(distutils.cmd.Command):
+    """Custom comand for updating the GeoJSON schemas on which this project depends."""
+
+    description = "Download and overwrite latest BOPTEST files from source"
+
+    user_options = [("baseurl=", "u", "base URL from which to download the schemas")]
+
+    def initialize_options(self):
+        self.files_to_download = {
+            'data/categories.json': 'alfalfa_worker/lib/data/categories.json',
```
Comment (nllong) [inline management/update_boptest.py:16]: initial list of files that we pull from BOPTEST. need to add others as we move along.

File: tests/integration/test_fmu_single_zone_vav.py [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+import datetime
+import json
+import os
+import pytest
+import sys
+import time
+from unittest import TestCase
+
+from alfalfa_client import AlfalfaClient, Historian
+from alfalfa_worker.lib.thermal_comfort import ThermalComfort
```
Comment (nllong) [inline tests/integration/test_fmu_single_zone_vav.py:12]: I'm thinking that ThermalComfort and units should get moved to alfalfa_client, right?

Comment (corymosiman12) [reply tests/integration/test_fmu_single_zone_vav.py:12]: I think it could be.  Can leave here for now

Comment (nllong) [reply tests/integration/test_fmu_single_zone_vav.py:12]: i'll move soon-ish. I want to get parity with BOPTEST next.

**Pull request merged**